src/Dumplie/Application/Command/Customer/AddToCart.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+declare (strict_types = 1);
+
+namespace Dumplie\Application\Command\Customer;
+
+final class AddToCart
+{
+    /**
+     * @var string
+     */
+    private $sku;
+
+    /**
+     * @var int
+     */
+    private $quantity;
+
+    /**
+     * @var string
+     */
+    private $cartId;
+
+    /**
+     * @param string $sku
+     * @param int    $quantity
+     * @param string $cartId
+     */
+    public function __construct(string $sku, int $quantity, string $cartId)
+    {
+        $this->sku = $sku;
+        $this->quantity = $quantity;
+        $this->cartId = $cartId;
```

Comment:
So here is how I see that: 
Commands are simple Data Transfer Objects, those objects are immutable and they reflect user requests. Those DTO are used to transfer data across different application layers (or event between different infrastructure elements like for example queues), in most cases they are created in UI layer and passed to application layer and then command handlers are using them to work on domain objects. Even if somebody will create AddToCart command with empty sku somehow (I think there should be UI validation for that anyway) then he will hit the wall because domain is validating itself and its protecting system from moving to unexpected state. 
IMO there is no need to add extra validation here, it's better to validate user inputs in UI layer (for example on forms and give nice looking feedback to user from there) instead of making those DTO larger then they already are.